### ISSUE TITLE
BlackBerry plugin (may) mess up the user's customized VS keyboard shortcuts?

### ISSUE THREAD

**Issue by HongkunWang** [phofman/vs-plugin#53]
I have my own customized Visual Studio keyboards (For example: Ctrl+A for SelectAll, Ctrl+Y for Redo) set. However, sometime after I install BB plugin, my keyboard shortcuts are gone. I re-assign my shortcuts but I lost them again from time to time.

I can not confirm this issue is caused by BB plugin, since I do have couple of other VS plugin installed meanwhile.

**Comment by phofman** [comment]
Nope, this plugin doesn't define any custom keyboard shortcuts. It only defines a top-level menu, which is accessible using `Alt+<key>` combination.

**Comment by HongkunWang** [comment]
Since the plugin is programmed by Visual Studio SDK, then there might be a chance that your plugin discard, bypass, or disable the loading of user-defined shortcuts.
I understand that this plugin does not define any keyboard shortcuts, but it may still has a chance.
Hongkun

**Comment by phofman** [comment]
Well, we could blame about the same any app that manipulates user's registry as they might delete settings directly from VS hive or any other VS extension. Without any evidence/logs or pointing to suspicious API call I wouldn’t say it's the fault of this plugin. The UI integration is done through XML file, so you should see a negative impact all the time.
